app/services/fetch_from_cache_service.rb

```diff
@@ -0,0 +1,26 @@
+class FetchFromCacheService
```

Comment:
I think this could use defining what it fetches from cache, just to differentiate it from say Rails.cache.fetch

Tricky thing is to define what they are, search associations?

Reply:
We ended up removing the `SearchResult` presenter (and therefore the code duplication), so we don't need this service anymore.